### DISCUSSION TITLE
左手用デバイスのランドが剥がれたと思われます。対処法をご教授願います。

### DISCUSSION THREAD

**Discussion by apio1969** [Yowkees/keyball#213] [Q&A] [answered]
お世話になります。はんだ付け初心者です。
表題の件、左手用として設定したデバイスのはんだ付けで、左手用デバイスとして使用する際に行うブリッジ作業（基板に「LEFT」
と書かれた場所です。）で何度か失敗して、ランドが剥がれたと思われます。対処の方法をご教授頂けると幸いです。
お手数をおかけしますが、よろしくお願いいたします。
![keyball61 Left](https://user-images.githubusercontent.com/130259394/230749368-94cfa43b-7c13-405a-85ef-373d19d0214b.jpg)

**Comment by Yowkees** [comment, marked as answer]
![Keyball61L_fix](https://user-images.githubusercontent.com/84209772/231066555-e0085d85-ebae-466f-8972-c3cae21b0d80.jpg)
ご連絡ありがとうございます。
添付画像に示した緑点の２箇所をエナメル線やゴム被覆導線でショートさせてください。
これで、LEFTジャンパを半田付けしたのと同じ効果になります。
お手数ですがご対応お願いいたします。

**Comment by apio1969** [comment]
ご教授、ありがとうございました。
早速ご指示の通りにやってみます。また不明点などがありましたらよろしくお願いいたします。

**Reply by apio1969** [reply]
お世話になります。
先日ご教授頂きましたとおり、銅線を買ってきてハンダでつなぎました。
うまく動作していなかった左手側デバイスが機能するようになりました。
白い銅線が売っていたので目立たなくすることができ、素人ながらなんとか
ハンダ付けもできたのでよかったです。
ありがとうございました。


![Uploading 20230413_195815.JPG…]()
